build.xml

```diff
@@ -50,7 +50,8 @@
             <arg path="src"/>
             <arg value="xml"/>
             <arg value="phpmd.xml"/>
-            <arg value="--exclude */Tests/*"/>
+            <arg value="--exclude"/>
```

Comment:
The diff shows `<arg value="--exclude */Tests/*"/>`, not `<arg line="--exclude */Tests/*"/>`. Note the difference between `line` and `value`, which are handled differently by Ant (although I'm not sure how it works exactly).

Anyway, I shouldn't have blocked the PR for it, it is just a suggestion.